devito/dle/backends/utils.py

```diff
@@ -12,7 +12,8 @@
     'par-region': lambda i: c.Pragma('omp parallel %s' % i),
     'par-for': c.Pragma('omp parallel for schedule(static)'),
     'simd-for': c.Pragma('omp simd'),
-    'simd-for-aligned': lambda i, j: c.Pragma('omp simd aligned(%s:%d)' % (i, j))
+    'simd-for-aligned': lambda i, j: c.Pragma('omp simd aligned(%s:%d)' % (i, j)),
```

Comment:
Is this now consistent with the `__attribute__((aligned`?

Reply:
they are different things. With ``__attribute__``, you're **enforcing** data alignment; with ``omp simd aligned`` you're just telling the compiler that "all datasets in this loop are aligned". While the former are necessary, the latter may actually be not, as long as the compiler is able to retrieve the information by itself (which is probably our case, but just to be sure, we put the ``aligned`` anyway)